packages/router/src/__typetests__/routeParamsTypes.test.ts

```diff
@@ -90,14 +90,12 @@ describe('RouteParams<>', () => {
     expect(globRoutes.description).type.toBeString()
   })
 
-  context('Glob params in the middle', () => {
-    test('Multiple Glob route params', () => {
-      const middleGlob: RouteParams<'/repo/{folders...}/edit'> = {
-        folders: 'src/lib/auth.js',
-      }
+  test('Glob params in the middle', () => {
+    const middleGlob: RouteParams<'/repo/{folders...}/edit'> = {
+      folders: 'src/lib/auth.js',
+    }
```

Comment:
@dac09 We had a conversation about this in #9394 already. I took a better look, seems like this is just a copy paste mistake.

The "Multiple Glob route params" test is defined above: https://github.com/redwoodjs/redwood/blob/b6457700abf209da0c23bfa8dc0fc3883f663dc2/packages/router/src/__typetests__/routeParamsTypes.test.ts#L67-L68

It tests multiple params indeed, but in the nested case only one param is tested. For me it looks that the nested "Multiple Glob route params" is redundant here. This test case is simply called "Glob params in the middle". Similar to "Starts with Glob route params" case above.

Reply:
Yeah agreed, looks that way to me too :)

Thank you for the PR again!